test/ao/sparsity/test_kernels.py

```diff
@@ -156,6 +156,7 @@ def test_sparse_qlinear(self):
                 model.qconfig = tq.get_default_qconfig('fbgemm')
                 qmodel = copy.deepcopy(model)
                 sqmodel = copy.deepcopy(model)
+                sqmodel.linear.mask = mask
```

Comment:
Not sure what happened to previous comments: This should be sqmodel.linear.fake_sparse.mask

Reply:
That's not e en supposed to be tested here -- this bled through from some other PR